package.json

```diff
@@ -22,7 +22,7 @@
     "fmt": "prettier --write .",
     "fmt.check": "prettier --check .",
     "lint": "eslint .",
-    "ngrok": "ngrok http http://localhost:5173",
+    "ngrok": "ngrok http --domain=hagfish-fond-honeybee.ngrok-free.app http://localhost:5173",
```

Comment:
Don't push your static domain, possibly the best solution would be to move that to environments

Reply:
Done